test/test_mp_reduce_scatter.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import torch
+import torch_xla.core.xla_model as xm
+import torch_xla.distributed.xla_multiprocessing as xmp
+
+
+def _mp_fn(index):
+  device = xm.xla_device()
+  world_size = xm.xrt_world_size()
+  scale = 1.0
```

Comment:
maybe it would be better to test the scale that is not 1.0?

Reply:
Thanks, done.